packages/knip/src/util/fs.ts

```diff
@@ -1,6 +1,7 @@
 import { readFileSync, statSync } from 'node:fs';
 import { readFile } from 'node:fs/promises';
 import yaml from 'js-yaml';
+import * as toml from 'smol-toml';
```

Comment:
Please use named imports whenever available

Reply:
I wasn't sure if importing `parse` would clutter the namespace.